src/libopensc/ctx.c

```diff
@@ -804,6 +804,29 @@ int sc_context_repair(sc_context_t **ctx_out)
 	return SC_SUCCESS;
 }
 
+#ifdef USE_OPENSSL3_LIBCTX
+int sc_openssl3_init(sc_context_t *ctx)
```

Comment:
If I see right, this is used only here, so I would prefer to keep it static.

Reply:
ok fixing this and the previous point given they are the same area of code